### ISSUE TITLE
Documentação

### ISSUE THREAD

**Issue by mm44m6** [idvogados/idvogados.github.io#18]
## Tarefa

Necessário melhorar documentação, colocando campos de como rodar o projeto e também como contribuir.

**Comment by ItanuRomero** [comment]
Concordo, se possível podemos fazer uma reunião entre o pessoal de TI, para que todos estejamos a par do projeto, vai economizar bastante tempo!
